### PR TITLE
Add methods to retrieve a storage API client

### PR DESCRIPTION
**Pull request by elharo**
@tgroh @chanseokoh Needed for #1745

### CODE REVIEW

File: plugins/com.google.cloud.tools.eclipse.googleapis.test/src/com/google/cloud/tools/eclipse/googleapis/internal/GoogleApiFactoryTest.java [hunk 1]
```diff
@@ -68,6 +70,12 @@ public void setUp() throws ExecutionException {
   }
 
   @Test
+  public void testNewStorageApi_Url() throws IOException, GoogleApiException {
```
Comment (chanseokoh) [inline plugins/com.google.cloud.tools.eclipse.googleapis.test/src/com/google/cloud/tools/eclipse/googleapis/internal/GoogleApiFactoryTest.java]: `IOException` not necessary.

File: plugins/com.google.cloud.tools.eclipse.googleapis/META-INF/MANIFEST.MF [hunk 2]
```diff
@@ -47,6 +47,7 @@ Import-Package: com.google.cloud.tools.eclipse.util,
  javax.servlet.http,
  org.codehaus.jackson,
  org.eclipse.core.net.proxy,
+ org.eclipse.osgi.storage,
```
Comment (chanseokoh) [inline plugins/com.google.cloud.tools.eclipse.googleapis/META-INF/MANIFEST.MF]: This is a different storage, isn't it?

File: plugins/com.google.cloud.tools.eclipse.googleapis/src/com/google/cloud/tools/eclipse/googleapis/internal/GoogleApiUrl.java [hunk 3]
```diff
@@ -19,7 +19,10 @@
 public enum GoogleApiUrl {
 
   APPENGINE_ADMIN_API("https://appengine.googleapis.com"),
-  CLOUDRESOURCE_MANAGER_API("https://cloudresourcemanager.googleapis.com");
+  CLOUDRESOURCE_MANAGER_API("https://cloudresourcemanager.googleapis.com"),
+  
+  // is this correct????
```
Comment (chanseokoh) [inline plugins/com.google.cloud.tools.eclipse.googleapis/src/com/google/cloud/tools/eclipse/googleapis/internal/GoogleApiUrl.java]: Well, docs I could find seem to say so.

File: plugins/com.google.cloud.tools.eclipse.googleapis.test/src/com/google/cloud/tools/eclipse/googleapis/internal/GoogleApiFactoryTest.java (hunk 1 above)
Comment (elharo) [inline plugins/com.google.cloud.tools.eclipse.googleapis.test/src/com/google/cloud/tools/eclipse/googleapis/internal/GoogleApiFactoryTest.java]: done

File: plugins/com.google.cloud.tools.eclipse.googleapis/META-INF/MANIFEST.MF (hunk 2 above)
Comment (elharo) [inline plugins/com.google.cloud.tools.eclipse.googleapis/META-INF/MANIFEST.MF]: done

File: plugins/com.google.cloud.tools.eclipse.googleapis/src/com/google/cloud/tools/eclipse/googleapis/internal/GoogleApiUrl.java (hunk 3 above)
Comment (elharo) [inline plugins/com.google.cloud.tools.eclipse.googleapis/src/com/google/cloud/tools/eclipse/googleapis/internal/GoogleApiUrl.java]: cool

**Pull request merged**